packages/wallet/src/services/ProviderTracker/TrackedAssetProvider.ts

```diff
@@ -19,12 +22,15 @@ export class AssetProviderStats {
  */
 export class TrackedAssetProvider extends ProviderTracker implements AssetProvider {
   readonly stats = new AssetProviderStats();
+  readonly healthCheck: AssetProvider['healthCheck'];
```

Comment:
Need to update tracked provider tests to include healthCheck

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/346/commits/96d73416672a64fecb97bf345ad1a70ed0a48c48